opentelemetry-sdk/CHANGELOG.md

```diff
@@ -42,6 +42,10 @@ also modified to suppress telemetry before invoking exporters.
   behind feature flag "experimental_metrics_custom_reader".
   [#2928](https://github.com/open-telemetry/opentelemetry-rust/pull/2928)
 
+- TODO: Placeholder for View related changelog. Polish this after all
```

Comment:
There will be another change in a followup PR to change Stream to use a proper builder pattern and return Result. the changelog will be finally updated after all changes are in.
@utpilla